jobs/ci-kubernetes-soak-cos-docker-validation-deploy.env

```diff
@@ -1,5 +1,8 @@
 KUBE_OS_DISTRIBUTION=gci
 
+# Use fixed k8s version.
+JENKINS_PUBLISHED_VERSION=v1.6.1
+
 ### soak-env
```

Comment:
> KUBE_GCE_MASTER_IMAGE=cos-docker-13

@kewu1992 Can you switch this job to use the `gci-next-canary` image family in `container-vm-image-staging` project instead?
I think you will need to replace `KUBE_GCE_MASTER_*` and `KUBE_GCE_NODE_*` vars below with something like:
```
JENKINS_GCI_HEAD_IMAGE_FAMILY=gci-next-canary
```

@wonderfly can confirm.